packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2581,6 +2581,9 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
     super.dispose();
   }
 
+  /// Delegate triggered when we remove the bottom insets
+  void Function(EdgeInsets edgeInsets)? onRemoveBottomInsets;
```

Comment:
also, this should really be a configuration option on the Scaffold, not a random property that anybody can set on the state.